src/requested-changes-download/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from "fs"
+import consistentSort from "../lib/consistentSort"
+import config from "./config"
+import convertRow from "./convertRow"
+import mergeRequests from "./mergeRequests"
+import SheetsClient from "./sheetsClient"
+
+export default async () => {
+  const allRows = await new SheetsClient(config).retrieveOffenceCodeRows()
```

Comment:
(nipick/suggestion) Shuold we add more console logs inbetween the different functions? might help in case lambda crashes and someone needs to debug this?

Reply:
good idea, I'll add some